cypress/e2e/ui/Overview/reports.cy.js

```diff
@@ -0,0 +1,431 @@
+/* eslint-disable no-undef */
+
+describe('Overview > Reports Tests', () => {
+  beforeEach(() => {
+    cy.viewport(1800, 800);
```

Comment:
Should we just make this the default viewport for all tests?

Reply:
Yeah we should. Should we do that in a separate pr though?

Reply:
@Fryguy I removed this line from this pr and fixed this in a new pr: https://github.com/ManageIQ/manageiq-ui-classic/pull/8893